docs/reference/datatiers.asciidoc

```diff
@@ -174,25 +212,58 @@ tier.
 [[data-tier-allocation]]
 === Data tier index allocation
 
-When you create an index, by default {es} sets
-<<tier-preference-allocation-filter, `index.routing.allocation.include._tier_preference`>>
+The <<tier-preference-allocation-filter, `index.routing.allocation.include._tier_preference`>> setting determines the tier index shards should be allocated to.
+
+When you create an index, by default {es} sets the `_tier_preference`
```

Comment:
@shainaraskas it might just be pedantic-ness catching me, but I feel we might be leaving a read-through logic gap that a "created index" could simultaneously be a data stream's backing index which'd behave differently. So this paragraph cannot be separated from the paragraph after because Elasticsearch's default `_tier_preference` is a two-step logic flow. 

Overall I don't otherwise have a problem with how this was previously written, but this is why I had collapsed the content. I don't feel strongly on what the phrasing is but (again maybe only pedantic and we can just ignore me but) I believe this should be a single paragraph.

Reply:
would saying "that isn't part of a data stream" help? the reason I'm chunking it a little is because if people are only looking for the second piece of info, it's not easy to scan to it.

Reply:
I'm going to side with this point being me feeling pedantic. I don't like it both/either way so no reason for it holding the conversation up 😂

Reply:
we can always pivot later if people are still having issues 👍